mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -176,12 +176,12 @@ def start_app(self, wait_time=APP_START_WAIT_TIME):
             '%s failed to start on %s.' % (self.app_name, self._adb.serial))
 
     def connect(self, port, addr='localhost', uid=UNKNOWN_UID,
-                connection_timeout=None, cmd=JsonRpcCommand.INIT):
+                cmd=JsonRpcCommand.INIT):
         """
         Opens a connection to the remote client.
 
         Opens a connection to a remote client. The connection will error out if
-        it takes longer than the connection_timeout time. Once connected if the
+        it takes longer than the _SOCKET_TIMEOUT seconds. Once connected if the
```

Comment:
This phrasing is a little confusing. Can you clarify that the second timeout applies to any other rpc?